erts/emulator/sys/unix/erl_child_setup.c

```diff
@@ -662,6 +665,21 @@ main(int argc, char *argv[])
     return 1;
 }
 
+static void kill_child(pid_t os_pid) {
+    if (os_pid > 0 && kill(os_pid, SIGTERM) != 0) {
+        DEBUG_PRINT("error killing process %d: %d", os_pid, errno);
```

Comment:
Should always work, but if it doesn't because of eg. a race condition between the child dying on its own and killing it, continue with trying to kill the other children.